### PR TITLE
Brief and Description for Editor Inspector

### PR DESCRIPTION
**Pull request by MostafaMTH**
Noticed there was not Brief Description nor Description for Editor Inspector, created one.

### CODE REVIEW

File: doc/classes/EditorInspector.xml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorInspector" inherits="ScrollContainer" version="4.0">
 	<brief_description>
+		A tab used to edit the properties of the selected.
 	</brief_description>
 	<description>
+		The editor inspector is by default located on the right-hand side of the editor. It is used to edit the properties of the node selected. For example, you can select a node such as the sprite2D then edit it's transform through the inspector tool. The editor inspector is an essential tool in the game development workflow.
```
Comment (aaronfranke) [inline doc/classes/EditorInspector.xml]: "A tab used to edit the properties of the selected" incomplete sentence, add the word "node".

"node selected" -> "selected node", "sprite2D" -> "Sprite2D" and remove "the", "it's -> its"

**Review by aaronfranke — CHANGES_REQUESTED** [review]

File: doc/classes/EditorInspector.xml (hunk 1 above)
Comment (MostafaMTH) [reply doc/classes/EditorInspector.xml]: Thanks for pointing this out, I will fix it.

Comment (MostafaMTH) [reply doc/classes/EditorInspector.xml]: Fixed the changes, can you please review it one more time?

File: doc/classes/EditorInspector.xml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorInspector" inherits="ScrollContainer" version="4.0">
 	<brief_description>
+		A tab used to edit the properties of the selected node.
 	</brief_description>
 	<description>
+		Editor inspector is by default located on the right-hand side of the editor. Its used to edit the properties of the selected node. For example, you can select a node such as the Sprite2D then edit it's transform through the inspector tool. Editor inspector is an essential tool in the game development workflow.
```
Comment (aaronfranke) [inline doc/classes/EditorInspector.xml]: You changed the wrong its. You need to make what is currently "its" into "it's" and what is currently "it's" into "its".

"Editor inspector" should have "The" in front of it in both instances.

**Review by aaronfranke — CHANGES_REQUESTED** [review]
There are still many grammar errors, you've already received 4 other reviews including the previous PRs that you closed: #35829 #35831

In addition to the grammar errors, I realize now that this might be completely missing the point. This documentation is for the class reference, therefore it should explain what you can do with the inspector through code. How can the signals below be interacted with, etc.

**Comment by MostafaMTH** [comment]
> There are still many grammar errors, you've already received 4 other reviews including the previous PRs that you closed: #35829 #35831
> 
> In addition to the grammar errors, I realize now that this might be completely missing the point. This documentation is for the class reference, therefore it should explain what you can do with the inspector through code. How can the signals below be interacted with, etc.

I am only doing the description not the methods or code

**Comment by MostafaMTH** [comment]
@aaronfranke May you please review the changes again?

File: doc/classes/EditorInspector.xml [hunk 3]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorInspector" inherits="ScrollContainer" version="4.0">
 	<brief_description>
+		A tab used to edit properties of the selected node.
 	</brief_description>
 	<description>
+		The editor inspector is by default located on the right-hand side of the editor. It's used to edit the properties of the selected node. For example, you can select a node such as the Sprite2D then edit its transform through the inspector tool. The ditor inspector is an essential tool in the game development workflow.
```
Comment (aaronfranke) [inline doc/classes/EditorInspector.xml]: "ditor" -> "editor"

Aside from that I think I'll approve this, since it's certainly better than nothing.

**Review by aaronfranke — CHANGES_REQUESTED** [review]

File: doc/classes/EditorInspector.xml (hunk 3 above)
Comment (MostafaMTH) [reply doc/classes/EditorInspector.xml]: Fixed it

**Review by aaronfranke — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.1.

**Pull request merged**